romana_setup.md

```diff
@@ -0,0 +1,117 @@
+# The `romana-setup` tool
+
+The `romana-setup` tool is used to provision and install Romana using the selected installation environment, linux distribution and stack type.
+
+## Usage Summary
+
+```
+romana-setup: executes the ansible playbooks for installing romana
+Usage: romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [action]
+       romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] <action> [ansible-options]
+       romana-setup <h|--help>
+
+Stack Name:  user-defined stack name (default: $USER)
+Platforms:   aws (default), vagrant
+Distro:      ubuntu (default), centos
+Stack Types: devstack (default), kubernetes
+Actions:     install (default), uninstall
+```
+
+## Option Details
+
+### Name
+
+A name for the installation environment can be specified using the `-n` or `--name` option.
```

Comment:
Might not be clear what an 'installation environment' is. I could think that this refers to my OS (Ubuntu?) or maybe my virtualenv, etc.

Maybe say something like: "You can give your cluster installation a name, so that you can refer to multiple clusters via the installation scripts." Or just call it "cluster name"?